### PR TITLE
Make it impossible that the last wimi leaves a project

### LINKED ISSUE #287: Projekt - Verlassen
1. Aktuell ist es möglich sich als letzter Wimi aus einem Projekt zu entfernen

**Erwartetes Verhalten**
- Es ist nicht möglich sich aus einem Projekt zu entfernen, wenn man der letzte Wimi dieses Projektes ist (dann gibt es die Button nicht, Links sollten auch nicht funktionieren)

### PR DESCRIPTION
**Pull request by arneboockmeyer**
fix #287

### CODE REVIEW

**Comment by WGierke** [comment]
LGTM ;)

**Comment by Zanthelo** [comment]
LGTM :see_no_evil:

**Comment by Tymolc** [comment]
LGTM :goat:

**Comment by NHoff95** [comment]
LGTM :+1:

File: spec/views/projects/show.html.erb_spec.rb [hunk 1]
```diff
@@ -96,10 +97,17 @@
 
     it 'shows leave project button if part of project' do
       @wimi.projects << @project
+      @wimi2.projects << @project
       visit current_path
       expect(page).to have_content('Leave Project')
     end
 
+    it 'dont show leave project button if he is the only wimi' do
```
Comment (bahnfahren) [inline spec/views/projects/show.html.erb_spec.rb]: vllt besser it 'does not show the leave ...'

**Pull request merged**